diff --git a/plugin/mikutter_mode/mikutter_mode.rb b/plugin/mikutter_mode/mikutter_mode.rb
index 2fa8372ce675..c4e2ea55403a 100644
--- a/plugin/mikutter_mode/mikutter_mode.rb
+++ b/plugin/mikutter_mode/mikutter_mode.rb
@@ -22,7 +22,8 @@ Plugin.create(:mikutter_mode) do
           notice "ruby code execute: \n#{file || code}"
           r = Server.main.instance_eval(code, file || "mikutter-mode onthefly executer")
           notice "returns: \n#{r.pretty_inspect}"
-          [r.pretty_inspect]
+          #[r.pretty_inspect]
+          [r.inspect]
         rescue Exception => e
           notice "exception:"
           notice e
